docs/build-for-developers/cli-usage.md

```diff
@@ -25,9 +30,8 @@ openfn deploy --help
 
 ### Run a job
 
-To run a single job, you must explicitly specify which adaptor to use. You can
-find the list of publicly available [adaptors here](/adaptors). See examples
-below.
+To run a single job, you must explicitly specify which adaptor to use.
```

Comment:
Can I suggest we remove `see examples below` and have a less terse link?

Maybe this:
```
To run a single job, you must explicitly specify which adaptor to use - see the [publicly available adaptors](/adaptors).
```